scripts/set-up-spack.sh

```diff
@@ -27,15 +27,14 @@ spack external find python perl binutils git tar xz bzip2
 spack config add 'packages:all:target:[x86_64]'
 # reuse as much as possible, make externals useful
 spack config add 'concretizer:reuse:true'
-# avoid building an external cmake and python
-spack config add 'packages:cmake:buildable:False'
-spack config add 'packages:python:buildable:False'
 # Generate spack environment for packages
 spack env create --dir /opt/env --with-view /opt/view
 
 # ensure clangs and others that don't inject rpaths make working executables
-echo /opt/view/lib > /etc/ld.so.conf.d/spack_view.conf
-echo /opt/view/lib64 > /etc/ld.so.conf.d/spack_view.conf
+cat >/etc/ld.so.conf.d/spack_view.conf <<EOF
```

Comment:
Could we have this in a re-usable script?

Reply:
We could, but it's part of the spack setup script already, are there other things you'd like to add to the load cache than the view?

Reply:
I don't understand the second part, but I'm good with just having this as long as it isn't duplicated in another file.

Reply:
The change there adds two paths to ld.so.conf so future binaries will search those paths for libraries, then updates the cache.  I'd be happy to put it in another script, but could only think of wanting to run the script if we wanted to add more paths or different paths or something.  If it's just for spack's view in opt then this should be the only place we need it.  As of now there's no duplication.